.github/workflows/enable-auto-merge.yaml

```
@@ -0,0 +1,23 @@
+
+permissions: {}
+
+jobs:
+  enable-auto-merge:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'dependencies')
+    permissions:
+      pull-requests: write
+    env:
+      PR_NUMBER: ${{ github.event.pull_request.number }}
+    steps:
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 18: did not find expected '-' indicator

(syntax-check)